apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue

```diff
@@ -138,14 +138,18 @@ const state = reactive({
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const isLatestVersion = (version: string) => true;
 
-const fetchCollectorUpdate = async (): Promise<CollectorModel> => SpaceConnector.client.inventory.collector.update({
-    collector_id: collectorFormStore.collectorId,
-    plugin_info: {
-        version: collectorFormState.version,
-        upgrade_mode: collectorFormState.autoUpgrade ? 'AUTO' : 'MANUAL',
-    },
-    tags: collectorFormState.tags,
-});
+const fetchCollectorUpdate = async (): Promise<CollectorModel> => {
+    if (!collectorFormStore.collectorId) throw new Error('collector_id is required');
+    const params: CollectorUpdateParameter = {
+        collector_id: collectorFormStore.collectorId,
+        plugin_info: {
+            version: collectorFormState.version,
+            upgrade_mode: collectorFormState.autoUpgrade ? 'AUTO' : 'MANUAL',
```

Comment:
suggestion: If there is a possibility that 'AUTO' or 'MANUAL' might be used elsewhere, it would be good to manage them as constants!